apps/myaccount/src/api/typingdna-data.ts

```diff
@@ -0,0 +1,64 @@
+import { IdentityClient } from "@wso2/identity-oidc-js";
+import { HttpMethods } from "@wso2is/core/models";
+import { store } from "../store";
+import * as https from "https";
+
+const httpClient = IdentityClient.getInstance().httpRequest.bind(IdentityClient.getInstance());
+
+
+/**
+ * This API is used to retrieve the QR code URL of the authenticated user.
+ */
+export const DeleteTypingPatterns = (): Promise<any> => {
```

Comment:
Use camel case function names. Also see if you can define the return type interface. (place it in the models folder)


ex: export const DeleteTypingPatterns = (): Promise<TypingPatternInterface> => {

```suggestion
export const deleteTypingPatterns = (): Promise<any> => {
```